rust/parquet/src/file/reader.rs

```diff
@@ -32,6 +32,22 @@ use crate::basic::Type;
 
 use crate::column::reader::ColumnReaderImpl;
 
+/// Parquet files must be read from end for the footer then from start for columns
+pub enum ChunkMode {
```

Comment:
One suggestion I have is to call this `ReadFrom` rather than `ChunkMode`to make the name more specific.

Alternatively, given its similarity, I wonder if it would make sense to use  `std::io::SeekFrom` here https://doc.rust-lang.org/std/io/enum.SeekFrom.html rather than a custom enum

Reply:
`std::io::SeekFrom` is similar but it has also the `SeekFrom::Current` state that we are not interested in. 
About the name, I agree that `ChunkMode` is not ideal, but I wanted it to explicitly relate to ChunkReader. What about the more verbose but probably also more explicit `ReadChunkFrom` ?

Reply:
I think`ReadChunkFrom` is reasonable. Or maybe `ChunkFrom` ?

Reply:
good idea